appvm-scripts/etc/securitylimits.d/90-qubes-gui.conf

```diff
@@ -1,4 +1,8 @@
-# Qubes GUI agent needs to mlock() all the composition buffers, so they are
-# constant physical address
-user   soft memlock 131072
-user   hard memlock 131072
+# Old Qubes GUI agent needs to mlock() all the composition buffers, so they are
+# constant physical addresses.  Current GUI agent needs to allocate large
+# amounts of pinned memory for grant sharing.
+@qubes   - memlock 131072
+
+# Qubes group is trusted and this makes audio handling easier
+@qubes   - nice -20
+@qubes   - rtprio unlimited
```

Comment:
Is it actually used by anything? Not sure about pipewire, but pulseaudio asks rtkit-daemon to change its priority, so this setting is not needed for it.

Reply:
I was having some sort of problem with rtkit, but this should be a separate PR.

Reply:
So, can you move this into separate PR, or drop completely until further diagnosed?

Reply:
ping?